src/include/stir/numerics/erfMapping.inl

```diff
@@ -0,0 +1,73 @@
+
+
+#include "stir/numerics/BSplines1DRegularGrid.h"
+
+START_NAMESPACE_STIR
+
+inline void
+erfMapping::
+setup()
+{
+  this->_sampling_period = (this->maximum_sample_value) / this->get_num_samples();
+  std::vector<double> erf_values;
+
+  //Compute a vector of erf values
+  for (int i=0; i<this->get_num_samples() ; ++i)
+    erf_values.push_back(erf(i * this->_sampling_period));
```

Comment:
why only store positive values? If you want to save time, an offset is probably better than branching with `if` below (probably very little difference though)